metadata-import-export/README.md

```diff
@@ -0,0 +1,42 @@
+# Doc tab
+
+## Description
+
+This tool is deployed as a tab inside FieldTwin Design. It shows how to respond to
+window messages, call the FieldTwin API, download and upload metadata using excel spreadsheet.
+
+This example queries FieldTwin for metadata attached to staggedAsset, connection or well using SheetJS library
+
+## Installation
+
+1. Self-hosted option: deploy the file `index.html` on a public or private web
+   hosting service and obtain the URL to reach it  
+   or  
+   Pre-hosted option: use the URL `https://xvisionas.github.io/FieldTwin-Integration-Demo/metadata-import-export/`
+2. In FieldTwin Admin, go to Account Settings, then Integrations, and click _Create New Tab_.
+   Set the following values:  
+   ```
+   Name:                           Docs Tab
+   URL:                            <the URL from step 1>
```

Comment:
In my demo, I had a problem that FieldTwin only allows https URLs and I couldn't see a way to use a http://localhost running demo?